source/euclid.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const gcd = (x, y) => {
+	while (x > 0 & y > 0) {
+		if (x > y) {
+			x = x % y;
+		}
+		else {
+			y = y % x;
+		}
+	}
+	
+	return x + y;
+}
+
+const euclid = (...numbers) => {
+	if (numbers.some((number) => !Number.isFinite(number) ? true : false)) {
+		return "Infinity number";
```

Comment:
```suggestion
		throw new TypeError("Not finite numbers");
```
плохой тон - лучше ошибку выбрасывать или null на крайняк